js/search-worker.js

```diff
@@ -2,38 +2,18 @@
 
 importScripts("/js/lunr.min.js");
 
-var data = {};
 var index;
 
-// Download the search data synchronously.
+// Download the search index synchronously.
 var req = new XMLHttpRequest();
 var async = false;
```

Comment:
Nit: Given that `async` is a keyword in TypeScript, which we use in a lot of places, perhaps `doAsync` would be a better variable name? Alternatively, since it is only used in one place, perhaps just `false /* async */` would be sufficient?

Reply:
Done.